src/Symfony/Component/EventDispatcher/Tests/DependencyInjection/RegisterListenersPassTest.php

```diff
@@ -247,6 +257,20 @@ public function testInvokableEventListener()
         $this->assertEquals($expectedCalls, $definition->getMethodCalls());
     }
 
+    public function testItThrowsAnExceptionIfTagIsMissingMethodAndClassHasNoValidMethod()
+    {
+        $this->expectException(InvalidArgumentException::class);
+        $this->expectExceptionMessage('Methods "onFooBar" and "__invoke" do not exist for service "foo". Please define the "method" attribute on "kernel.event_listener" tags.');
```

Comment:
Maybe better like this.
```suggestion
        $this->expectExceptionMessage('None of the "onFooBar" or "__invoke" methods exist for the service "foo". Please define the "method" attribute on "kernel.event_listener" tags.');
```

Reply:
:pray: